web/modules/custom/dpl_something_similar/src/Controller/DplSomethingSimilarController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drupal\dpl_something_similar\Controller;
+
+use Drupal\Core\Block\BlockManagerInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Render\RendererInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * Render something similar react app.
+ */
+class DplSomethingSimilarController extends ControllerBase {
+
+  /**
+   * DplSomethingSimilarController constructor.
+   *
+   * @param \Drupal\Core\Block\BlockManagerInterface $blockManager
+   *   Drupal block manager.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   Drupal renderer service.
+   */
+  public function __construct(
+    private BlockManagerInterface $blockManager,
+    private RendererInterface $renderer
+  ) {
+  }
```

Comment:
Is this intentionally empty if so then please document it

Reply:
Yes, the promoter pattern

Reply:
Check